Enterprise-CMCS/managed-care-review
# [MCR-2540] state notified when submitting a response

## Summary

This PR sends an email to the state contacts and submitters for a submission when a response is sent back to CMS

- `sendQuestionResponseStateEmail.ts` created
- `sendQuestionResponseStateEmail.eta` template created
- `createQuestionResponse.ts` resolver updated to call the `sendQuestionResponseStateEmail.ts` emailer function
- `sendQuestionStateEmail.ts` updated to receive a `currentQuestion` instead of an array of `questions` since it only needs info from the most recent question submitted

#### Related issues
https://qmacbis.atlassian.net/browse/MCR-2540

#### Screenshots

<img width="1150" alt="Screenshot 2023-12-11 at 4 09 32 PM" src="https://github.com/Enterprise-CMCS/managed-care-review/assets/67110378/2e379ecf-d6db-42b6-9bf3-c4e944662dfb">

#### Test cases covered

- the to address list goes to the expected emails which are the state contacts and submission submitters
- no duplicate email addresses
- subject line is correct
- the email itself includes relevant info (like link to a submission, question round number, pkg and rate names, division of the CMS user who submitted the question)
- renders overall email


## QA guidance

- Log in as a CMS user and submit a question
- Log in as a state user and submit a response to the question
- Check the inbox and ensure the email is as expected
